src/templates/Author.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+const Author = () => {
+    <div>
+        alex is dumb
```

Comment:
Please don't commit jokes, this commit history is visible to everyone. Memes are fair game in slack and git reviews tho